packages.yml

```diff
@@ -2,5 +2,5 @@
 packages:
 - package: fivetran/fivetran_utils
   version: [">=0.4.0", "<0.5.0"]
-# - package: dbt-labs/spark_utils
-#   version: [">=0.3.0", "<0.4.0"]
\ No newline at end of file
+# - package: dbt-labs/spark_utils 
+#   version: [">=0.3.0", "<0.4.0"] 
```

Comment:
Is this intentionally left out?

Reply:
Oh I had commented it out because I wasn't sure if it was necessary in the source package since it seemed to run fine without it on databricks. Should I leave it in by default?

Reply:
Yeah let's leave it in as I assume the transform package will require it and we can keep this dependency in this package.

Reply:
Done.